src/pages/Promotion/Content.js

```diff
@@ -25,111 +28,189 @@ export default class Content extends React.Component {
                             dealing with medical record
                         </div>
                         <div className="part1-btn">
-                            <Button type="primary" shape="round">Get Started</Button>
+                            <Button type="primary" shape="round">TRY DR.CHAIN FOR FREE</Button>
+                            <Button type="primary" shape="round">LEARN MORE</Button>
                         </div>
                     </Col>
-                    <Col span="6"></Col>
+                    <Col span="10"></Col>
                 </Row>
-                <Row>
-                    <Col className="page-transient"/>
+                <Row className="part2">
+                    <Col xs={1} sm={1} md={2} lg={2} xl={3} xxl={3}/>
+                    <Col xs={22} sm={22} md={20} lg={20} xl={18} xxl={18}>
+                        <img src="/assets/images/carousel.png" alt="carousel"></img> 
```

Comment:
1. The left image seems changed, could we use the latest one?
2. Is there any way we can improve the quality of this image? It seems quite blur on my browser.

Reply:
needs to be changed in the figma. Will tell minz do so